File: Clase-2/Actividad-En-Clase/Interpretado/ejercicio1.py
import random

def ejercicio1():
    # Ingrese 10 valores por teclado. Presentar la suma y promedio
    valores = []
    for i in range(10):
        valor = float(input(f"Ingrese el valor {i+1}: "))
        valores.append(valor)
    
    suma = sum(valores)
    promedio = suma / len(valores)
    
    print(f"Suma: {suma}")
    print(f"Promedio: {promedio}")

ejercicio1()

File: Clase-2/Actividad-En-Clase/Compilado/ejercicio1.cpp
#include <iostream>
#include <vector>
#include <random>
#include <chrono>
#include <iomanip>

void ejercicio1() {
    std::vector<double> valores(10);
    double suma = 0;

    for (int i = 0; i < 10; ++i) {
        std::cout << "Ingrese el valor " << i + 1 << ": ";
        std::cin >> valores[i];
        suma += valores[i];
    }

    double promedio = suma / valores.size();

    std::cout << "Suma: " << suma << std::endl;
    std::cout << "Promedio: " << promedio << std::endl;
}


int main() {
    ejercicio1();

    return 0;
}